payment_methods/SquareOnsite/help_tabs/pm_overview_square.help_tab.php

```diff
@@ -53,12 +53,35 @@
     </li>
     <li>
         <strong><?php esc_html_e('Enable Digital Wallet', 'event_espresso'); ?></strong><br />
+        <?php esc_html_e(
+            'Allows payments with digital wallets like Apple Pay, Google Pay.',
```

Comment:
since there's only two items listed, we should use "and" instead of a comma, ie:

```suggestion
            'Allows payments with digital wallets like Apple Pay and Google Pay.',
```